client/src/pages/Availability/Availability.tsx

```diff
@@ -0,0 +1,297 @@
+import { Button, InputLabel, FormControl } from '@mui/material';
+import { Box } from '@mui/system';
+import SettingHeader from '../../components/SettingsHeader/SettingsHeader';
+import { Typography } from '@mui/material';
+import Select from '@mui/material/Select';
+import MenuItem from '@mui/material/MenuItem';
+import CreateAvailInterface from './AvailInterface';
+import { Formik, FormikHelpers } from 'formik';
+import React, { useState, useEffect } from 'react';
+import FormInput from '../../components/FormInput/FormInput';
+import CreateSchedule from '../../helpers/APICalls/createSchedule';
+import { useAuth } from '../../context/useAuthContext';
+import { useSnackBar } from '../../context/useSnackbarContext';
+import GetSchedule from '../../helpers/APICalls/getSchedules';
```

Comment:
```suggestion
import getSchedule from '../../helpers/APICalls/getSchedules';
```
Functions names in JS use camelCase

Reply:
Done the changes

Reply:
Done the changes